### ISSUE TITLE
A11y tool is missing "type" from it's attribute list

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#253]

**Comment by Rafi993** [comment]
do you mean role of the element ?

**Comment by argyleink** [comment]
role is already tracked https://github.com/GoogleChromeLabs/ProjectVisBug/blob/master/app/utilities/design-properties.js#L28

type, like `type=number` or `type=email`, which are valuable types on inputs for usability and accessibility 👍
